app/pb_kits/playbook/pb_nav/_item.jsx

```diff
@@ -1,30 +1,55 @@
 /* @flow */
 
 import React from 'react'
+import { Icon } from '../'
 
 type NavItemProps = {
   text: String,
   link: String,
+  iconLeft: String,
+  iconRight: String,
   active?: Boolean,
   children: React.Node,
+  onClick?: EventHandler,
 }
 
 const NavItem = ({
   text = '',
-  link = '',
+  link,
   active = false,
+  iconLeft,
+  iconRight,
   children,
-}: NavItemProps) => (
-  <li className={`pb_nav_list_border_item${active ? '_active' : ''}`}>
-    <a
-        className="pb_nav_list_item_link"
-        href={link}
-    >
-      <span className="pb_nav_list_item_text">
-        {text || children}
-      </span>
-    </a>
-  </li>
-)
+  onClick = () => {},
+}: NavItemProps) => {
+  const Tag = link ? 'a' : 'div'
 
+  return (
+    <li className={`pb_nav_list_kit_item${active ? '_active' : ''}`}>
+      <Tag
+          className="pb_nav_list_item_link"
+          href={link}
+          onClick={onClick}
```

Comment:
@thestephenmarshall I added an onclick here since this kit didn't have one. Thoughts?